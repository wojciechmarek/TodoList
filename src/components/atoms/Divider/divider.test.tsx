describe('Divider', () => {
  it('should render', () => {
    expect(true).toBeTruthy();
  });

  describe('when the button is clicked', () => {});
});
