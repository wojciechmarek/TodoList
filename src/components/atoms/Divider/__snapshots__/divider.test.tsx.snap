// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider should match the snapshot 1`] = `
.emotion-0 {
  height: 1px;
  box-shadow: var(--divider-box-shadow);
}

<div
  className="emotion-0"
/>
`;
